src/material/table/table.scss

```diff
@@ -1,5 +1,5 @@
 $mat-header-row-height: 56px;
-$mat-row-height: 48px;
+$mat-row-height: 51px;
```

Comment:
We should have a comment about why it's 51px, even though the spec says 52px (due to the border).